src/sonic-framework/rebootbackend/reboot_thread.cpp

```diff
@@ -0,0 +1,264 @@
+#include "reboot_thread.h"
+#include <google/protobuf/util/json_util.h>
+#include <chrono>
+#include "dbconnector.h"
+#include "logger.h"
+#include "notificationproducer.h"
+#include "reboot_common.h"
+#include "reboot_interfaces.h"
+#include "redis_utils.h"
+#include "select.h"
+#include "selectableevent.h"
+#include "selectabletimer.h"
+#include "subscriberstatetable.h"
+#include "system/system.pb.h"
+#include "timestamp.h"
+
+namespace rebootbackend {
+
+using namespace ::gnoi::system;
+using steady_clock = std::chrono::steady_clock;
+using Progress = ::rebootbackend::RebootThread::Progress;
+//using WarmBootStage = ::swss::WarmStart::WarmBootStage;
```

Comment:
Here as well. If not use let's just remove it.

Reply:
Pl. refer to the latest PR https://github.com/sonic-net/sonic-buildimage/pull/20713 where its removed